config/runtime.config.ts

```diff
@@ -5,7 +5,9 @@ export default defineNuxtConfig({
             API_URL           : process.env.NUXT_PUBLIC_API_URL           || 'https://api.cbddev.xyz',
             API_DIRECT_URL    : process.env.NUXT_PUBLIC_API_URL           || 'https://api-direct.cbd.int',
             REALM_CONF_HOST   : process.env.NUXT_PUBLIC_REALM_CONF_HOST   || 'ort.cbddev.xyz',
-            CAPTCHA_V2_BADGE_KEY: process.env.CAPTCHA_V2_BADGE_KEY        || '*******',
+            CAPTCHA_V2_BADGE_KEY: process.env.NUXT_PUBLIC_CAPTCHA_V2_BADGE_KEY        || '*******',
+            SITE_ALERT        : '',
+            SITE_ALERT_LEVEL  : process.env.NUXT_PUBLIC_SITE_ALERT_LEVEL || 'danger',
```

Comment:
nuxt3 sonfig does not need to passe process.env.NUXT_PUBLIC_SITE_ALERT_LEVEL.
```suggestion
            siteAlert:'',
            siteAlertLevel : 'danger',
```
you can use camelcalse also with default value :
eg: